project/Dependencies.scala

```diff
@@ -53,6 +53,27 @@ object Dependencies {
       "com.typesafe.akka" %% "akka-http-testkit"                  % AkkaHttpVersion % "test"
     )
 
+    // often called-in transitively with insecure versions of databind / core
+    val JacksonDatabind = Seq(
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.9.7"
```

Comment:
I have confirmed that the pom.xml of a `publishLocal` contains:
```
        <dependency>
            <groupId>com.fasterxml.jackson.core</groupId>
            <artifactId>jackson-databind</artifactId>
            <version>2.9.7</version>
        </dependency>
```

Reply:
Thanks for checking.